app.rb

```diff
@@ -66,6 +66,15 @@ def fixture(path)
       fixture 'acuant/get_results_response.json'
     end
 
+    # LexisNexis TrueID
+    post '/restws/identity/v3/accounts/:account_number/workflows/:workflow_name/conversations' do
+      case params[:workflow_name]
+      when /TrueID/
+        sleep ENV['LEXISNEXIS_TRUE_ID_DELAY'].to_f
+        fixture 'lexisnexis/true_id_response.json'
+      end
```

Comment:
Don't need the `case` ?

```suggestion
    post '/restws/identity/v3/accounts/:account_number/workflows/TrueID/conversations' do
      sleep ENV['LEXISNEXIS_TRUE_ID_DELAY'].to_f
      fixture 'lexisnexis/true_id_response.json'
```

Reply:
There are multiple TrueID workflows, so this was intended to support that.  It could switch to an `if` though?

Reply:
Ah, I think it's fine to keep as-is then.